plugins/PrivacyManager/templates/gdprOverview.twig

```diff
@@ -14,6 +14,8 @@
         </p>
     </div>
 
+    {{ postEvent('Template.afterGDPROverviewIntro') }}
```

Comment:
Is this supposed to be in this PR? Seems unrelated

Reply:
Yep it was supposed to be, forgot to mention it... didn't want to create a new PR for it :)